scripts/scripts.js

```diff
@@ -157,6 +155,22 @@ const loadStyle = (path) => {
   eagerLoad(marquee.querySelector('img'));
 }());
 
+export const [setLibs, getLibs] = (() => {
```

Comment:
Do we have to export here? A little concerned about side effects from self-executing functions if someone actually imported it.